server/licences/licenceStatus.ts

```diff
@@ -72,6 +72,16 @@ const statusConfig: Record<LicenceStatus, LicenceStatusConfig> = {
     description: 'Outside pilot area',
     colour: 'grey',
   },
+  OOS_BOTUS: {
+    label: 'Breach of supervision',
```

Comment:
Is this text too long for the badge? How does it look?

Reply:
It is 21 chars, same as `Variation in progress`.